docs/Overview.md

```diff
@@ -0,0 +1,409 @@
+# @remoteoss/remote-flows
+
+## Overview
+
+Welcome to the `@remoteoss/remote-flows` package, a React library that provides components for Remote's embbeded solution.
+
+## Installation
+
+```sh
+npm install @remoteoss/remote-flows
+```
+
+## Getting Started
+
+After installation, import the main CSS file in your application:
+
+```css
+@import '@remoteoss/remote-flows/index.css';
+```
+
+### Basic Setup
+
+#### Cost calculator rendering
+
+```tsx
+import { CostCalculator, RemoteFlows } from '@remoteoss/remote-flows';
+import './App.css';
+
+const estimationOptions = {
+  title: 'Estimate for a new company',
+  includeBenefits: true,
+  includeCostBreakdowns: true,
+};
+
+function CostCalculatorForm() {
+  return (
+    <CostCalculator
+      estimationOptions={estimationOptions}
+      defaultValues={{
+        countryRegionSlug: 'a1aea868-0e0a-4cd7-9b73-9941d92e5bbe',
+        currencySlug: 'eur-acf7d6b5-654a-449f-873f-aca61a280eba',
+        salary: '50000',
+      }}
+      params={{
```

Comment:
I didn't realise about `params` prop but I'm not a big fan of this name. I think we call this `options` and add jsdocs to disclaimer -> label describing what it does. It's hard to get it, just by looking at it. This can be address in a separate PR though 👍